src/engine/objects/gonerchoice.lua

```diff
@@ -29,7 +29,10 @@ function GonerChoice:init(x, y, choices, on_complete, on_select)
 
     self.soul = Sprite("player/heart_blur")
     self.soul:setScale(2, 2)
-    self.soul:setColor(1, 0, 0)
+    self.soul:setColor(Kristal.getSoulColor())
+	if MainMenu.mod_list:getSelectedMod().soulColor then	-- Add Game detection
```

Comment:
This'll trigger if you're in game, not just the menu, won't it?

Reply:
No, because you're immediately overwriting it with the `setColor` in the if statement.

Reply:
oh shit you're right, that's a thing I forgot to add, hang on a sec